src/index.ts

```diff
@@ -108,7 +110,13 @@ export const args = parse<CLIArgs>(
 			optional: true,
 			description: cliArgsDescription.plan
 		},
-		confDir: { type: String, alias: 'd', optional: true }
+		confDir: { type: String, alias: 'd', optional: true },
+		inspect: {
+			type: Boolean,
+			alias: 'i',
+			defaultValue: false,
+			optional: true
+		}
```

Comment:
Keep Formatting same as #L113
```suggestion
		inspect: { type: Boolean, alias: 'i', defaultValue: false, optional: true }
```